docs/features/secrets/consume-from-key-vault.md

```diff
@@ -7,7 +7,7 @@ layout: default
 You can easily create a Key Vault secret provider - The only thing you need to do is specify how you want to configure and to what vault.
 
 ```csharp
-var vaultAuthentication = new ManagedServiceIdentityAuthentication();
+var vaultAuthentication = new ManagedServiceIdentityAuthenticator();
```

Comment:
In older versions, this was indeed the case, (see older docs) but this is the current docs of the most recently released version, where this class was renamed. See [this](https://github.com/arcus-azure/arcus.security/blob/master/src/Arcus.Security.Providers.AzureKeyVault/Authentication/ManagedServiceIdentityAuthentication.cs)

Reply:
okay - I had to upgrade to the preview version for this.  will roll back the above change